README.md

```diff
@@ -1,25 +1,24 @@
-# SEO: A plugin for Pelican :fr:
+# SEO Plugin for Pelican :fr:
 
 [![Build Status](https://img.shields.io/github/workflow/status/pelican-plugins/seo/build)](https://github.com/pelican-plugins/seo/actions)
 
-This plugin helps you improve your Pelican site SEO (Search Engine Optimization) to reach the top positions on search engines. To see what can do SEO for you, go directly to [Usage](#usage) section.
+This plugin helps you improve your Pelican site SEO (Search Engine Optimization) to reach the top positions on search engines. To see what this SEO plugin can do for you, visit the [Usage](#usage) section.
 
 | Author          | GitHub                            |
 | :-------------: | :-------------------------------: |
 | Maëva Brunelles | https://github.com/MaevaBrunelles |
 
-## Why do you need SEO
+## Why
+
+You might want to use this plugin if:
 
-If :
-* you need some help to improve the SEO of your Pelican site
 * you are not familiar with SEO
-* you want a simple site with all SEO basis
```

Comment:
Hi Maeva. I suggested removing this line because I don't understand its value. Is this plugin only for simple sites? What if my site is not simple? And what is "SEO basis"? Perhaps this line could have value with different wording, but since I don't understand it, it's difficult for me to propose alternative phrasing.

Reply:
I think there is a misunderstanding with the word "simple", but maybe that's because I'm not an English-native speaker By this word, I wanted to mean "unmindfully", "without bothering". About "SEO basis", in FR we used this term to represent the essentials SEO optimizations for a website . 
In a nutshell, it can be summerized as follows: "thanks to this plugin, you can have a site with SEO essentials improvments which allow it to be more easily indexed by the search engines, without bothering", but I think we can find a prettier shorter version.

Reply:
That helps me better understand the intention. Thank you. I changed that bullet point to:
> • you want an easy way to apply SEO fundamentals

Reply:
This is perfect!